browser/app/js/components/Browse.js

```diff
@@ -497,338 +508,346 @@ export default class Browse extends React.Component {
-            <span className="la-actions pull-right"><button onClick={ this.downloadSelected.bind(this) }> Download all as zip </button></span>
-            <span className="la-actions pull-right"><button onClick={ this.showDeleteConfirmation.bind(this) }> Delete selected </button></span>
-            <i className="la-close fa fa-times" onClick={ this.clearSelected.bind(this) }></i>
-          </div>
-          <Dropzone>
-            { alertBox }
-            <header className="fe-header-mobile hidden-lg hidden-md">
-              <div id="feh-trigger" className={ 'feh-trigger ' + (classNames({
-                                                  'feht-toggled': sidebarStatus
-                                                })) } onClick={ this.toggleSidebar.bind(this, !sidebarStatus) }>
-                <div className="feht-lines">
-                  <div className="top"></div>
-                  <div className="center"></div>
-                  <div className="bottom"></div>
-                </div>
+      <section className="browser__inner">
+        { alertBox }
+        <section className={ classNames({
+                               'content': true,
+                               'content--toggled': sidebarStatus
+                             }) }>
+          <header className="header">
+            <div className="toolbar">
+              <div className="actions">
+                <button className="zmdi zmdi-menu" onClick={ this.toggleSidebar.bind(this, !sidebarStatus) } />
+                <button className="zmdi zmdi-view-comfy" />
+                <button onClick={ this.showDeleteConfirmation.bind(this) } disabled={ checkedObjects.length == 0 } className="zmdi zmdi-delete" />
+                { deleteButton }
+                <button onClick={ this.downloadSelected.bind(this) } disabled={ checkedObjects.length == 0 } className="zmdi zmdi-download" />
+              </div>
+              { loginButton }
+              { browserDropdownButton }
+            </div>
+            <Path selectPrefix={ this.selectPrefix.bind(this) } />
+            <BrowserUpdate />
+          </header>
+          <SideBar searchBuckets={ this.searchBuckets.bind(this) }
+            selectBucket={ this.selectBucket.bind(this) }
+            clickOutside={ this.hideSidebar.bind(this) }
+            showPolicy={ this.showBucketPolicy.bind(this) }
+            storageDetails={ storageUsageDetails } />
+          <div className="objects">
+            <header className="objects__row" data-type="folder">
+              <div className="objects__item objects__item--name" onClick={ this.sortObjectsByName.bind(this) } data-sort="name">
+                Name
+                <i className={ classNames({
+                                 'objects__item__sort': true,
+                                 'zmdi': true,
+                                 'zmdi-sort-desc': sortNameOrder,
```

Comment:
It seems to me like the `AZ` icon (and maybe the others?) should be displayed closer to Name rather than Size when the table header is hovered over. What do you think? I thought the icon was associated with Size, maybe because there isn't a border between each. Here's a screenshot.
![screen shot 2017-07-31 at 5 02 40 pm](https://user-images.githubusercontent.com/3038254/28803643-01d968d6-7613-11e7-9029-2e077cf225c5.png)